### PR TITLE
Enable huge services in tests with less than 1k nodes

### PR DESCRIPTION
**Pull request by jupblb**

### CODE REVIEW

File: config/jobs/kubernetes/sig-scalability/sig-scalability-periodic-jobs.yaml [hunk 1]
```diff
@@ -244,6 +246,8 @@ periodics:
       - --kubemark-nodes=100
       - --provider=gce
       - --metadata-sources=cl2-metadata.json
+      - --env=CL2_ENABLE_HUGE_SERVICES=true
+      - --env=CL2_ENABLE_HUGE_SERVICES_MULTIPLE_REPLICAS=true
```
Comment (marseel) [inline config/jobs/kubernetes/sig-scalability/sig-scalability-periodic-jobs.yaml:250]: It's not load test.

File: config/jobs/kubernetes/sig-scalability/sig-scalability-periodic-jobs.yaml [hunk 2]
```diff
@@ -322,6 +326,8 @@ periodics:
       - --env=CONTROLLER_MANAGER_TEST_ARGS=--profiling --kube-api-qps=300 --kube-api-burst=300
       - --env=SCHEDULER_TEST_ARGS=--profiling --kube-api-qps=300 --kube-api-burst=300
       - --env=CL2_ENABLE_CLUSTER_OOMS_TRACKER=true
+      - --env=CL2_ENABLE_HUGE_SERVICES=true
+      - --env=CL2_ENABLE_HUGE_SERVICES_MULTIPLE_REPLICAS=true
```
Comment (marseel) [inline config/jobs/kubernetes/sig-scalability/sig-scalability-periodic-jobs.yaml:330]: It's not load test.

File: config/jobs/kubernetes/sig-scalability/sig-scalability-periodic-jobs.yaml (hunk 1 above)
Comment (jupblb) [reply config/jobs/kubernetes/sig-scalability/sig-scalability-periodic-jobs.yaml]: Fixed.

File: config/jobs/kubernetes/sig-scalability/sig-scalability-periodic-jobs.yaml (hunk 2 above)
Comment (jupblb) [reply config/jobs/kubernetes/sig-scalability/sig-scalability-periodic-jobs.yaml]: Fixed.

**Pull request merged**